content/millennium/dstu2/scheduling/slot.md

```diff
@@ -73,12 +73,9 @@ _Implementation Notes_
 
 Notes:   
 
+- At least one of the `slot-type`, `schedule.actor`, and `-location` parameters must be provided.
 - The `slot-type`, `schedule.actor`, and `-location` parameters may be included only once.
```

Comment:
There needs to be language that clarifies that slot-type can only be included once, but a comma separated series of slot-types is allowed now.